qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1097,6 +1097,16 @@ def predecessors(self, node):
         """Returns iterator of the predecessors of a node as DAGNodes."""
         raise NotImplementedError()
 
+    def quantum_successors(self, node):
```

Comment:
Since this extends the current `DAGCircuit` API, I suggest to add a test in `test/python//dagcircuit/test_dagcircuit.py`